### ISSUE TITLE
VRBangers posters not downloading

### ISSUE THREAD

**Issue by d2dyno** [PAhelper/PhoenixAdult.bundle#123]
Was about halfway through matching this set when posters ceased to download. not sure if it's a change on their side or rate limiting. There are no errors in the log:
```
2019-03-07 01:19:12,909 (1457186a4700) :  INFO (__init__:45) - *******MEDIA TITLE****** Vrbangers Pop My Corn 3dv
2019-03-07 01:19:12,909 (1457186a4700) :  INFO (__init__:52) - Getting Search Settings for: Vrbangers Pop My Corn 3dv
2019-03-07 01:19:12,913 (1457186a4700) :  INFO (PAsearchSites:698) - ^^^^^^^ siteID: 474
2019-03-07 01:19:12,913 (1457186a4700) :  INFO (PAsearchSites:700) - ^^^^^^^ Shortening Title
2019-03-07 01:19:12,913 (1457186a4700) :  INFO (PAsearchSites:701) - vrbangers vs vrbangers
2019-03-07 01:19:12,913 (1457186a4700) :  INFO (PAsearchSites:704) - 1
2019-03-07 01:19:12,913 (1457186a4700) :  INFO (PAsearchSites:717) - 6
2019-03-07 01:19:12,913 (1457186a4700) :  INFO (PAsearchSites:751) - searchTitle (before date processing): Pop My Corn 3dv
2019-03-07 01:19:12,914 (1457186a4700) :  INFO (__init__:62) - Search Title: Pop My Corn 3dv
2019-03-07 01:19:12,914 (1457186a4700) :  INFO (__init__:71) - Pop%20My%20Corn%203dv
2019-03-07 01:19:12,916 (1457186a4700) :  DEBUG (networking:166) - Requesting 'https://vrbangers.com/?post_type=video&s=Pop%20My%20Corn%203dv'
2019-03-07 01:19:13,668 (1457186a4700) :  DEBUG (networking:166) - Requesting 'https://vrbangers.com/video/pop-my-corn/'
2019-03-07 01:19:14,874 (1457186a4700) :  DEBUG (networking:166) - Requesting 'https://vrbangers.com/video/my-dirty-secret/'
2019-03-07 01:19:16,143 (1457186a4700) :  DEBUG (networking:166) - Requesting 'https://vrbangers.com/video/my-uber-hookup/'
2019-03-07 01:19:17,554 (1457186a4700) :  DEBUG (networking:166) - Requesting 'https://vrbangers.com/video/your-personal-trainer/'
2019-03-07 01:19:18,667 (1457186a4700) :  DEBUG (networking:166) - Requesting 'https://vrbangers.com/video/too-much-cumfidence/'
2019-03-07 01:19:20,018 (1457186a4700) :  DEBUG (runtime:88) - Sending packed state data (944 bytes)
2019-03-07 01:19:20,019 (1457186a4700) :  DEBUG (runtime:924) - Response: [200] str, 2340 bytes
2019-03-07 01:19:25,025 (1457186a4700) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNQoyMzcxNXM0CmRiaWRpMApzNwp2ZXJzaW9ubnMxMApwYXJlbnRHVUlEbnM4CnBhcmVudElEczUKTW92aWVzMTAKbWVkaWFfdHlwZXM4Ngpjb20ucGxleGFwcC5hZ2VudHMucGhvZW5peGFkdWx0Oi8vaHR0cHM6X192cmJhbmdlcnMuY29tX3ZpZGVvX3BvcC1teS1jb3JuX3w0NzQ%40bGFuZz1lbnM0Cmd1aWRzNDQKaHR0cHM6X192cmJhbmdlcnMuY29tX3ZpZGVvX3BvcC1teS1jb3JuX3w0NzRzMgppZHIwCg__
2019-03-07 01:19:25,026 (1457186a4700) :  DEBUG (runtime:49) - Received packed state data (912 bytes)
2019-03-07 01:19:25,027 (1457186a4700) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlVwZGF0ZU1ldGFkYXRh/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMTAKczIKZW5zNApsYW5nYjFzNQpmb3JjZWIwczgKcGVyaW9kaWNzNQoyMzcxNXM0CmRiaWRpMApzNwp2ZXJzaW9ubnMxMApwYXJlbnRHVUlEbnM4CnBhcmVudElEczUKTW92aWVzMTAKbWVkaWFfdHlwZXM4Ngpjb20ucGxleGFwcC5hZ2VudHMucGhvZW5peGFkdWx0Oi8vaHR0cHM6X192cmJhbmdlcnMuY29tX3ZpZGVvX3BvcC1teS1jb3JuX3w0NzQ@bGFuZz1lbnM0Cmd1aWRzNDQKaHR0cHM6X192cmJhbmdlcnMuY29tX3ZpZGVvX3BvcC1teS1jb3JuX3w0NzRzMgppZHIwCg__
2019-03-07 01:19:25,028 (1457186a4700) :  DEBUG (model:32) - Loading model with GUID com.plexapp.agents.phoenixadult://https:__vrbangers.com_video_pop-my-corn_|474?lang=en
2019-03-07 01:19:25,029 (1457186a4700) :  DEBUG (model:234) - Deserializing from /config/Library/Application Support/Plex Media Server/Metadata/Movies/8/e99607d02336428f87d922bba7edf5760da722d.bundle/Contents/com.plexapp.agents.phoenixadult/Info.xml
2019-03-07 01:19:25,036 (1457186a4700) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/23715/tree'
2019-03-07 01:19:25,043 (1457186a4700) :  INFO (__init__:664) - ******UPDATE CALLED*******
2019-03-07 01:19:25,043 (1457186a4700) :  INFO (__init__:667) - 474
2019-03-07 01:19:25,044 (1457186a4700) :  DEBUG (networking:166) - Requesting 'https://vrbangers.com/video/pop-my-corn/'
2019-03-07 01:19:26,396 (1457186a4700) :  INFO (siteVRBangers:52) - genre: Big Dick
2019-03-07 01:19:26,396 (1457186a4700) :  INFO (siteVRBangers:52) - genre: Blowjob
2019-03-07 01:19:26,396 (1457186a4700) :  INFO (siteVRBangers:52) - genre: Brunette
2019-03-07 01:19:26,396 (1457186a4700) :  INFO (siteVRBangers:52) - genre: Cowgirl
2019-03-07 01:19:26,397 (1457186a4700) :  INFO (siteVRBangers:52) - genre: Cum Swallow
2019-03-07 01:19:26,397 (1457186a4700) :  INFO (siteVRBangers:52) - genre: Cum-shot
2019-03-07 01:19:26,397 (1457186a4700) :  INFO (siteVRBangers:52) - genre: Deepthroat
2019-03-07 01:19:26,397 (1457186a4700) :  INFO (siteVRBangers:52) - genre: Natural Tits
2019-03-07 01:19:26,397 (1457186a4700) :  INFO (siteVRBangers:52) - genre: Shaved Pussy
2019-03-07 01:19:26,397 (1457186a4700) :  INFO (siteVRBangers:52) - genre: Skinny
2019-03-07 01:19:26,398 (1457186a4700) :  INFO (siteVRBangers:52) - genre: Small Tits
2019-03-07 01:19:26,398 (1457186a4700) :  INFO (siteVRBangers:52) - genre: Tattoo
2019-03-07 01:19:26,398 (1457186a4700) :  INFO (siteVRBangers:52) - genre: Teen
2019-03-07 01:19:26,398 (1457186a4700) :  INFO (siteVRBangers:61) - actor: Megan Rain
2019-03-07 01:19:26,399 (1457186a4700) :  DEBUG (networking:166) - Requesting 'https://vrbangers.com/model/megan-rain/'
2019-03-07 01:19:27,783 (1457186a4700) :  INFO (__init__:1030) - Genres
2019-03-07 01:19:27,785 (1457186a4700) :  INFO (__init__:1035) - Actors
2019-03-07 01:19:27,785 (1457186a4700) :  INFO (PAactors:67) - Actor: Megan Rain https://cdn.vrbangers.com/wp-content/uploads/2017/06/Megan-Rain-500x750.jpg
2019-03-07 01:19:27,787 (1457186a4700) :  DEBUG (model:229) - Serializing to /config/Library/Application Support/Plex Media Server/Metadata/Movies/8/e99607d02336428f87d922bba7edf5760da722d.bundle/Contents/com.plexapp.agents.phoenixadult/Info.xml
```

**Comment by PAhelper** [comment]
I only had one file from this site. I'm downloading more... for science.

**Comment by PAhelper** [comment]
None of the ones I downloaded had any art, so if some of yours were working at first, then something must have changed. Anyway, this is fixed in db43f8d33b7fd74e1574fd150c923c503345ff9b
